### ISSUE TITLE
linux run installer, accept license with argument

### ISSUE THREAD

**Issue by fjallraven** [Autodesk/maya-usd#2681]
It would be great if we could auto accept Autodesk license with an argument "--confirm"
If this is related: https://github.com/megastep/makeself/pull/100
It looks to have gone into makeself 2.3.1

Many thanks

**Comment by tanant** [comment]
@fjallraven - I'm guessing that this would be for pipeline/undattended deployment scenarios?

**Comment by fjallraven** [comment]
> @fjallraven - I'm guessing that this would be for pipeline/undattended deployment scenarios?

Yes exactly that. I could not find any workaround, example extracting the rpm etc, but if you have any I'll be happy to try it!

**Comment by fjallraven** [comment]
Thank you @santosg87 
That works well!

**Comment by santosg87** [comment]
glad to help! :)

moving this issue to the backlog for the time being as we discuss viability of fixing somewthing like this on the installer in the future.
